### ISSUE TITLE
`FAMD` methods not implented

### ISSUE THREAD

**Issue by ksuchak1990** [MaxHalford/prince#139]
Thanks for this really cool package. I was just looking at using `FAMD` and noticed that whilst it inherits from `PCA`, it has a number of methods that are marked as not implented. Is there a reason why these cannot simply use the inheritted method?

I was looking specifically at `FAMD.column_contributions_()` as I was looking to find out what the components comprise of when running `FAMD` - would the corresponding method used to calculate this in `PCA` not work here?

**Comment by MaxHalford** [comment]
Regarding column contributions, the formula should be this:

```py
import prince

famd = prince.FAMD(engine='scipy')
famd = famd.fit(dataset)
print(100 * (famd.column_coordinates_ ** 2).div(famd.eigenvalues_, axis=1))
```

| variable                       |            0 |          1 |
|:-------------------------------|-------------:|-----------:|
| alcohol_by_volume              | 22.8037      |  2.87668   |
| international_bitterness_units | 17.6866      | 11.0453    |
| standard_reference_method      |  9.1155      | 29.0644    |
| final_gravity                  | 22.9156      |  1.51193   |
| style_Adambier                 |  0.000313396 |  0.0140282 |

However, if you compare to FactoMineR, the results are not exactly the same:

```py
from rpy2.robjects import r as R
import tempfile

R("library('FactoMineR')")
with tempfile.NamedTemporaryFile() as fp:
    dataset.to_csv(fp)
    R(f"dataset <- read.csv('{fp.name}', row.names=c(1))")
    R(f"famd <- FAMD(dataset, graph=F)")

R("print(famd$var$contrib)")
```

```
                                     Dim.1      Dim.2      Dim.3      Dim.4
alcohol_by_volume              22.82992806  2.8240014  1.6980270  7.6644349
international_bitterness_units 17.60286532 11.9346353  8.9506370  1.3021757
standard_reference_method       9.31864159 31.6353908  3.9453317  0.3633442
final_gravity                  22.79472499  1.2868578  6.7993487  4.7912109
is_organic                      0.04318325  0.1163922  0.1178607  1.0851433
style                          27.41065679 52.2027224 78.4887949 84.7936910
```

So there is some work to do to understand what FactoMineR is doing. I can't find any paper which clearly explains what the correct formula is.

**Comment by ksuchak1990** [comment]
Thank @MaxHalford, I see your point. 

I had a look at the corresponding part of `FactoMineR` (https://github.com/husson/FactoMineR/blob/master/R/FAMD.R) and found it pretty hard to follow, so similarly can't tell what the _correct_ approach would be.

**Comment by MaxHalford** [comment]
I can't make a hard promise, but I will try to take a look this week, depending on my workload :)

**Comment by MaxHalford** [comment]
I've figured out the `column_coordinates`. I'll keep digging for `column_contributions` :)

**Comment by MaxHalford** [comment]
Ok I've implemented it! Just released version 0.9 🎉

Please let me know if you want other methods implemented. I'm mostly going to develop Prince based on what people need it for.
